Make test_no_replace_quant_permute_dequant_with_requantize use GraphBuilder.

eigen-k opened the pull request:
Summary: Use GraphBuilder to create the model for unit testing.

Differential Revision: D74842294

mcremon-meta commented on backends/cadence/aot/tests/test_fusion_ops_passes.py:
```diff
@@ -281,25 +281,23 @@ def forward(self, x):
+        )
+        builder.output(dequant)
+        graph_module = FuseQuantDequantToRequantizePass(
+            force_quant_dequant_fusion=False
```
any reason to add the flag here?

eigen-k replied:
Although the parameter defaults to False, I chose to set it explicitly to False to avoid any ambiguity, given that the adjacent test has it set to True. Does it seem redundant?

mcremon-meta replied:
I have no issue with it, was just wondering if there was a reason, but I noticed the other test being True after this review :)

eigen-k committed:
Make test_no_replace_quant_permute_dequant_with_requantize use GraphBuilder. (#10927)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/10927

Use GraphBuilder to create the model for unit testing.

Reviewed By: mcremon-meta, zonglinpeng

Differential Revision: D74842294
